### PR TITLE
add max accel and decel

### LINKED ISSUE #59: Add max acceleration and deceleration

### PR DESCRIPTION
**Pull request by eshedmargalit**
Would close #59

These work by computing discrete acceleration values between pairs of velocity calculations (which themselves are discrete approximations), then retaining either the max (fastest accel) or the min (hardest decel)

### CODE REVIEW

File: src/components/ActivityMap/ActivityStats.tsx [hunk 1]
```diff
@@ -55,6 +57,28 @@ export function ActivityStats({
               units="min/mi"
             />
           </Grid>
+          <Grid>
+            <Statistic
+              name="🏃‍♂️ Max Acceleration"
+              value={
+                activity.records
```
Comment (aradmargalit) [inline src/components/ActivityMap/ActivityStats.tsx:64]: Why not lift this to the `<Grid />`?

```tsx
{activity.records && <Grid> ... }
```

File: src/components/ActivityMap/ActivityStats.tsx [hunk 2]
```diff
@@ -86,6 +110,20 @@ export function ActivityStats({
                 units="min/mi"
               />
             </Grid>
+            <Grid>
+              <Statistic
+                name="🐶 Max Acceleration"
+                value={computeMaxAccel(garminActivity.records).toFixed(1)}
```
Comment (aradmargalit) [inline src/components/ActivityMap/ActivityStats.tsx:116]: Any noticable perf issues running this in the render function? Wonder if this is expensive enough to memoize

**Review by aradmargalit — APPROVED** [review]

File: src/components/ActivityMap/ActivityStats.tsx (hunk 2 above)
Comment (aradmargalit) [reply src/components/ActivityMap/ActivityStats.tsx:116]: I tested this, it's plenty fast

File: src/components/ActivityMap/ActivityStats.tsx (hunk 1 above)
Comment (eshedmargalit) [reply src/components/ActivityMap/ActivityStats.tsx:64]: Good idea, done!

**Review by aradmargalit — APPROVED** [review]

**Pull request merged**